docs/modules/ROOT/partials/proc-registry-operator-quickstart-sr.adoc

```diff
@@ -5,22 +5,21 @@ You can quickly deploy the {operator} on the command line, without the Operator
 
 .Prerequisites
 
-* You must go to link:{LinkRedHatIntegrationDownloads}[{NameRedHatIntegrationDownloads}], select the product version, and download the {registry} CRDs `.zip` file.
+* You must go to link:{LinkRedHatIntegrationDownloads}[{NameRedHatIntegrationDownloads}], select the product version, and download the {registry} examples `.zip` file.
```

Comment:
Suggest `...and download the examples in the {registry} CRDs `.zip` file.`

(The link on the Customer Portal is called *{registry} 2.0.0 CRDs*. Unfortunately, there is no mention of examples in the link text, so it's hard to know which link you need unless you spell it out clearly)

Reply:
I believe that administrator access is not needed if the operator is already installed.